.goreleaser.yaml

```
@@ -14,6 +14,10 @@ builds:
     goarch:
       - amd64
       - arm64
+    ldflags:
+      - -s -w
+      - -X github.com/mackerelio-labs/check-mackerel-metric/checkmackerelmetric.version={{ .Version }}
+      - -X github.com/mackerelio-labs/check-mackerel-metric/checkmackerelmetric.revision={{ .ShortCommit }}
```

[FYI] It might be possible to eliminate redundancy by referencing the environment variables defined in builds.

```
builds:
  - env:
      - CGO_ENABLED=0
      - PACKAGE_PATH=github.com/mackerelio-labs/check-mackerel-metric/checkmackerelmetric
```

```suggestion
      - -X {{ .Env.PACKAGE_PATH }}.revision={{ .ShortCommit }}
```

If goreleaser release --snapshot --clean succeeds locally, it will probably work.

---

It looks smart, thanks!